spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe CommentsController, type: :controller do
+  let!(:post) { FactoryGirl.create(:post) }
+
+  describe 'GET #new' do
+    login_user
+    before { get :new, post_id: post }
+
+    it { is_expected.to render_template 'new'}
+
+    it 'assigns the post to @post' do
+      expect(assigns :post).to eq post
```

Comment:
Style/NestedParenthesizedCalls: Add parentheses to nested method call `assigns :post`.